### PR TITLE
Ma/authorisation

### PR DESCRIPTION
**Pull request by mdacoca**
### Motivation and context

-- Authorisation page updated: text and screenshots

### Checklist

- [X] I submit my changes into the `develop` branch
- [] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [X] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: .github/workflows/full.yml [hunk 1]
```diff
@@ -13,7 +13,6 @@ on:
 
 env:
   WORKFLOW_RUN_URL: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
-  CYPRESS_VERIFY_TIMEOUT: 180000 # https://docs.cypress.io/guides/guides/command-line#cypress-verify
```
[the remark this answers is not in the document]
Comment (mdacoca) [reply .github/workflows/full.yml:16]: Sure

File: README.md [hunk 2]
```diff
@@ -215,7 +215,6 @@ questions and get our support.
 
 ## Links
 
-- [Computer Vision Annotation Tool (CVAT) has a New Home](https://opencv.org/opencv-and-cvat-computer-vision-annotation-tool-intel/)
```
Comment (nmanovic) [inline README.md:218]: @mdacoca , again. The change is mine. How have you removed it?

Comment (mdacoca) [reply README.md:218]: When I create a branch from dev it downloads a lot of extra stuff including some things from other branches and commits (including some of my commits and some not mine). So some of them are reverted some not. I try to figure out why it is this way.

Comment (mdacoca) [reply README.md]: @nmanovic I will redo it completly, sorry for the mess

**Pull request closed without merging**